pkg/activator/net/revision_backends.go

```diff
@@ -158,6 +158,11 @@ func (rw *revisionWatcher) probeClusterIP(dest string) (bool, error) {
 // probePodIPs will probe the given target Pod IPs and will return
 // the ones that are successfully probed, or an error.
 func (rw *revisionWatcher) probePodIPs(dests []string) (sets.String, error) {
+	// Short circuit case where dests == healthyPods
+	if rw.allDestsHealthy(dests) {
```

Comment:
`if reflect.DeepEquals(a, b)`?
or `if cmp.Equal()` -- it is guaranteed not to panic here.

Reply:
Actually `cmp.Equal` is going to be much slower, so `reflect.DeepEqual`